src/Momento.Sdk.Incubating/Requests/CollectionUpdateTtl.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+namespace Momento.Sdk.Incubating.Requests
+{
+    public record struct CollectionUpdateTtl(TimeSpan? ttl = null, bool refreshTtl = true)
+    {
+        public static CollectionUpdateTtl noRefreshOnUpdates()
+        {
+            return new CollectionUpdateTtl(ttl: null, refreshTtl: false);
+        }
+        public static CollectionUpdateTtl fromTimespan(TimeSpan ttl)
```

Comment:
To follow the`noRefreshOnUpdates` semantics, how about `refreshOnUpdatesFromTimeSpan`?